hafta1/hafta1.md

```diff
@@ -609,11 +609,29 @@ Bu yüzden lütfen dışarıdan bir uygulama kuracaksınız, forumlara ya da you
 
 ---
 
+## AUR(Arch User Repository) Nedir ?
+
+AUR, archlinux topluluğunun geliştirdikleri uygulama paketlerini, kütüphaneleri ve daha bir çok şeyi tüm archlinux topluluğuyla paylaştığı repodur. AUR da bulunan paketler ana repoda yoktur. AUR da bulunan paketler topluluk tarafından oylanarak olumlu yönde oy almış paketler ana repoya taşınabilir.
+
+## AUR Helpers
+
+Ana repodan bir paket indirmek istediğimizde komut satırına pacman yazıyoruz. Peki Arch User reposundan bir paket indirmek istediğimizde ne kullanmamız gerekiyor ?
+Burada devreye AUR helpers giriyor. AUR içinde bir paketi aramak yada indirmek için helpersları kullanırız. 
+- `aura`
+- `pacaur`
+- `pakku`
+- `pikaur`
+- `trizen`
+- `yay(yet another yogurt)`
+- `yaourt`
+
+Yukarıda bazı AUR helperlar listelenmiştir. Bunlardan yaourt ve trizen günümüzde daha çok tercih edilen 2 paket yöneticisidir. Biz burada trizeni kullanacağız çünkü yaourt'a göre daha güvenli ve kullanımı basit. Sizinde bunu kullanmanızı tavsiye ediyoruz :)
```

Comment:
Bu ifade yanlış olmuş. En çok kullanılan ikisi demeyelim. Diğer kısımlar güzel. Bu bağlantıya atıfta bulunalım yaourt'un güvenliği ile ilgili.
https://www.reddit.com/r/archlinux/comments/4azqyb/whats_so_bad_with_yaourt/

Aur helper seçimi için bu bağlantıya atıfta bulunalım
https://wiki.archlinux.org/index.php/AUR_helpers